lib/utils/overwriteWebpackConfig.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+function overwriteWebpackConfig(config) {
+  // we use InfrastructureLogger for logger and it respects stats.colors
+  if (config[0].stats == null) {
+    config[0].stats = {};
+    config[0].stats.colors = true;
+  } else if (typeof config[0].stats === 'object') {
+    if (config[0].stats.colors == null) {
+      config[0].stats.colors = true;
+    }
+  }
+}
+
+module.exports = overwriteWebpackConfig;
```

Comment:
I don't think we should override `stats.colors`

Reply:
We should respect original value

Reply:
many products won't show colors because they don't set stats.colors.(default is enabled at v4) 
So I overwrite it to keep between v4 and v5.

https://github.com/webpack/webpack-dev-server/blob/75718b7e25a27da598340cb00e23628d9496cd1a/bin/options.js#L76

Reply:
@hiroppy In webpack@5 it is `true` by default? Maybe we should use this logic inside webpack

Reply:
no, so I overwrite colors to stats.colors.

Reply:
I think we should not override `colors`, because it can break output in some terminals

Reply:
ok so what's your solution?

Reply:
Get this options from stats, maybe we should improve this options by default on webpack side using logic from `supported-color` package